src/fear_and_greed/cnn.py

```diff
@@ -62,3 +53,24 @@ def get(fetcher: Fetcher = None) -> FearGreedIndex:
         description=response["rating"],
         last_update=datetime.datetime.fromisoformat(response["timestamp"]),
     )
+
+def historical(fetcher: Fetcher = None) -> dict:
+    """Returns CNN's Fear & Greed Index historical data."""
+
+    if fetcher is None:
+        fetcher = Fetcher()
+
+    response = fetcher()["fear_and_greed_historical"]
+    fear_greed_historical = []
+    
+    if "data" in response:
+        historical_data = response["data"]
+        for data in historical_data:
+            fear_greed_historical.append(FearGreedIndex(
+                value=data["y"],
+                description=data["rating"],
+                last_update=datetime.datetime.fromtimestamp(data["x"] / 1000),
+            ))
+    
+    return fear_greed_historical
```

Comment:
Are the values in `response["data"]` guaranteed to be sorted by date?

Consider sorting them here to return the response is deterministic.